src/components/browser-integration.tsx

```diff
@@ -392,17 +360,17 @@ const BrowserFunctions = () => {
         console.log(body);
```

Comment:
Is this `console.log` needed?

Reply:
yup, it helps me knowing the contents of computed `body`.